src/pat/modal/modal.js

```diff
@@ -11,6 +11,8 @@ define([
     parser.addArgument("closing", ["close-button"], ["close-button", "outside"], true);
     parser.addArgument("close-text", 'Close');
     parser.addArgument("panel-header-content", ":first:not(.header)");
+    parser.addArgument("url", null);  // inject-url
+    parser.addArgument("trigger", null);  // inject-trigger
```

Comment:
I was thinking of naming these arguments ``inject-url`` and ``inject-trigger`` but found that this is confusing. Other opinions?